CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to wolfi-dev/os
+
+* [Setup development environmment](#setup-development-environment)
+
+## Setup development environment
+
+To ease the development of Wofli OS, you can use `chainguard-images/sdk` container image that already include both [apko](https://github.com/chainguard-dev/apko) & [melange](https://github.com/chainguard-dev/melange).
+
+To start a development environment, do
+
+```sh
+make dev-container
+```
+
+What it does is start the `chainguard-images/sdk` image and mount the current working directory into it.
+
+Now, the `Makefile` script assume that there're melange folder & melange binary in there. I change it locally to the following because it's where they are in `chainguard-image/sdk`.
+
+```
+MELANGE_DIR ?= /usr/share/melange
+MELANGE ?= $(shell which melange)
+```
+
+## Write your first Wolfi package
+
+Wofie is using melange to build packages. If you want to learn the details what commands are used when building packages, they are in the [`ci-build.yaml`](.github/workflows/ci-build.yaml) and [Makefile](Makefile).
```

Comment:
I think this will change at some point, but I reworked the phrase a little bit:

```suggestion
Wolfi packages are built using melange. If you want to learn how packages are built, you can see all the details in the [`ci-build`](.github/workflows/ci-build.yaml) workflow and in the [Makefile](Makefile).
```